src/blenderbim/blenderbim/bim/module/group/operator.py

```diff
@@ -130,7 +130,7 @@ def execute(self, context):
             new.name = attribute.name()
             new.is_null = data[attribute.name()] is None
             new.is_optional = attribute.optional()
-            new.string_value = "" if new.is_null else data[attribute.name()]
+            new.set_value("" if new.is_null else data[attribute.name()])
```

Comment:
Same, use `import_attributes` - nothing special here, it can work without any special callbacks.